src/components/categories.js

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 
-function categories() {
+function Categories() {
   return (
     <div>
       <button type="button">Check status</button>
```

Comment:
Your project looks great. Kindly ensure that all actions are dispatched in the corresponding components. The "Check Status" button is expected to have a function in the component which dispatches the "underConstruction" action creator when the button is clicked.